content/neuvector-prime/azure/contents.lr

```diff
@@ -137,7 +186,7 @@ Home > Cost Management <subscription> | Cost analysis
 
 ## Upgrading a NeuVector PAYG Cluster
 
-The Azure marketplace PAYG offer is tied to a billing adapter AND Neuvector version. These are updated periodically as new version of the billing adapter or NeuVector are released. To update the NeuVector version to the latest version supported by the marketplace listing, use the "az k8s-extension update" command.
+The Azure marketplace PAYG offer is periodically updated as a new versions of Neuvector are released, and for optimizations in the integration with Azure. To update the latest version of the Neuvector PAYG offering supported in the marketplace listing, use the "az k8s-extension update" command.
 
 Execute the following command in Cluster Cloud Shell:
```

Comment:
maybe give an example of going from version x.x.x to x.x.y ?

Reply:
also i think they need to register to use the command 

# Set your subscription.
az account set --subscription <your-subId>
# Register for preview features.
az feature register --namespace Microsoft.KubernetesConfiguration --name extensionTypes

Reply:
Updated the following content


As a Prerequisites to create extension instances in AKS cluster,  Register Azure Feature

# Set your subscription.
az account set --subscription <your-subId>
# Register for preview features.
az feature register --namespace Microsoft.KubernetesConfiguration --name extensionTypes

To check the state is registered

az feature list -o table --query "[?contains(name, 'Microsoft.ContainerService/Microsoft.KubernetesConfiguration')].{Name:name,State:properties.state}"


> maybe give an example of going from version x.x.x to x.x.y ?

# Update Example (Refer screenshot)
az k8s-extension update  --name testnv2-plan --cluster-name testnv2 --resource-group testnv-res-grp --cluster-type managedClusters --version 1.0.51